### DISCUSSION TITLE
Tectonic support

### DISCUSSION THREAD

**Discussion by berceanu** [roaldarbol/LaPreprint#21] [General]
Does this template work with [tectonic](https://tectonic-typesetting.github.io/en-US/)? Is so, what is the recommended workflow?

**Comment by roaldarbol** [comment]
I don't think it currently does. It's been a while since I tried tectonic (tried to make it work with VS Code), and I won't have the time for a while. If you have the time and can create a neat source structure that works, I'd be happy to add it to the wiki. 😊
